### DISCUSSION TITLE
How can I blur Ellipse Control by avalonia?

### DISCUSSION THREAD

**Discussion by kyuranger** [AvaloniaUI/AvaloniaVS#320] [Q&A]
Previously I did this by WPF like this:
`
<Ellipse Width="100" Height="100" Fill="Blue">
    <Ellipse.Effect>
        <BlurEffect Radius="5"/>
    </Ellipse.Effect>
</Ellipse>
`

Whereas, Avalonia has not Effect property. How can I blur the control?

**Comment by maxkatz6** [comment]
Duplicate of https://github.com/AvaloniaUI/Avalonia/issues/7518
